CMakeLists.txt

```diff
@@ -139,6 +139,16 @@ IF (NOT "${CMAKE_VERSION}" VERSION_LESS "2.8.2")
         ENDIF ()
 
     ENDIF ( CMAKE_SYSTEM MATCHES Linux )
+    
+    IF ( "${packageManagement}" STREQUAL "DEB" )
```

Comment:
Can we set the version as "${HPCC_MAJOR}.${HPCC_MINOR}.${HPCC_POINT}", then append the '~' if DEB?